File: launchdarkly/php/src/GetAIConfigVariationExample.php
<?php

namespace OSEG\LaunchDarklyExamples;

require_once __DIR__ . '/../vendor/autoload.php';

use SplFileObject;
use LaunchDarkly;

$config = LaunchDarkly\Client\Configuration::getDefaultConfiguration();
$config->setApiKey("ApiKey", "YOUR_API_KEY");

try {
    $response = (new LaunchDarkly\Client\Api\AIConfigsBetaApi(config: $config))->getAIConfigVariation(
        ld_api_version: null,
        project_key: "default",
        config_key: "default",
        variation_key: "default",
    );

    print_r($response);
} catch (LaunchDarkly\Client\ApiException $e) {
    echo "Exception when calling AIConfigsBetaApi#getAIConfigVariation: {$e->getMessage()}";
}

File: launchdarkly/kotlin/src/main/kotlin/oseg/launchdarkly_examples/GetAIConfigVariationExample.kt
package oseg.launchdarkly_examples

import com.launchdarkly.client.infrastructure.*
import com.launchdarkly.client.apis.*
import com.launchdarkly.client.models.*

import java.io.File
import java.time.LocalDate
import java.time.OffsetDateTime
import kotlin.collections.ArrayList
import kotlin.collections.List
import kotlin.collections.Map
import com.squareup.moshi.adapter

@ExperimentalStdlibApi
class GetAIConfigVariationExample
{
    fun getAIConfigVariation()
    {
        ApiClient.apiKey["ApiKey"] = "YOUR_API_KEY"

        try
        {
            val response = AIConfigsBetaApi().getAIConfigVariation(
                lDAPIVersion = null,
                projectKey = "default",
                configKey = "default",
                variationKey = "default",
            )

            println(response)
        } catch (e: ClientException) {
            println("4xx response calling AIConfigsBetaApi#getAIConfigVariation")
            e.printStackTrace()
        } catch (e: ServerException) {
            println("5xx response calling AIConfigsBetaApi#getAIConfigVariation")
            e.printStackTrace()
        }
    }
}
